pom.xml

```diff
@@ -63,6 +63,12 @@
             <artifactId>postgresql</artifactId>
             <version>42.7.0</version>
         </dependency>
+
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
```

Comment:
Not a call to action, just a notion. Jackson is a little bit more popular than gson and is used in popular frameworks like Spring